demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -112,10 +117,10 @@ def main():
     args = build_arg_parser().parse_args()
 
     # Loading source image
-    img = cv2.imread(args.input, cv2.IMREAD_COLOR)
-    if img is None:
-        print("Cannot load image " + args.input)
-        return -1
+    cap = open_images_capture(args.input, False)
+    image = cap.read()
+    if image is None:
+        raise RuntimeError("Can't read an image from the input")
```

Comment:
The demo works with only one image, but `DIrReader` may have multiple. No need to update this demo. The same is for `monodepth_demo`, later we may update its pipeline to work with multiple images.